registration.php

```diff
@@ -0,0 +1,36 @@
+<?php include('includes/header.php');?>
+<div id="page-container">
+
+    <div id="content-wrap">
+
+        <div class="container">
+
+            <h1>Register Today</h1>
+            <hr>
+
+            <form action="" method="POST">
+
+                <label>First Name:</label><br>
```

Comment:
For each `<label>` make sure to add the for attribute with the value set with the id of the input that it's connected to. For example this one would be: `<label for="first_name">`